sycl/test-e2e/AddressSanitizer/out-of-bounds/USM/parallel_for_int.cpp

```diff
@@ -16,7 +16,7 @@
 
 int main() {
   sycl::queue Q;
-  constexpr std::size_t N = 1234567;
+  constexpr std::size_t N = 512;
```

Comment:
is this line of change a nfc, or does the change make the test pass on dg2?

Reply:
yes, dg2 can't run too large workgroup